packages/client/hmi-client/src/workflow/ops/model-from-document/tera-model-from-doucment-drilldown.vue

```diff
@@ -0,0 +1,462 @@
+<template>
+	<tera-drilldown :title="node.displayName" @on-close-clicked="emit('close')">
+		<div>
```

Comment:
Why the `<div>`?

Reply:
The way the `tera-drilldown` component currently laid out, if there is that each child under the drilldown would be a new tab panel.  

i.e.
```
<tera-drilldown>
   <div tabName=Wizard>
   ...section
   </div>
   <div tabName=Notebook>
   ...section
   </div>
```

Since there is only one tab we need to surround it with a div (or any other html tag) to let the tera drilldown know that we only have one tab panel rather than it split every child div into another tab